### DISCUSSION TITLE
How to stateCount all tag data in one panel,  one tag one stateCount time series?

### DISCUSSION THREAD

**Discussion by yoursaaron** [grafana/grafana#64091] [Explore]
measuremet like this: _time tag=err_code field=errCode
data:
t1 err_code=1 errCode = 1
t2 err_code=2 errCode = 2
t3 err_code=1 errCode = 1
t4 err_code=2 errCode = 2
...

How to stateCount all tag data in one panel,  one tag one stateCount time series.

**Comment by yoursaaron** [comment]
result like this:
![stateCount](https://user-images.githubusercontent.com/20594735/222621684-ce13e94c-d275-4a82-b80e-45326eaf5ffb.png)

**Comment by Elfo404** [comment]
Closing this as we use GH discussions for discussing and track feature requests. please refer to our [community forum](https://community.grafana.com/) for support/questions
